index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>HTML and CSS Quiz</title>
+	<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300italic,600' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+
+	<!-- CARD ONE -->
+	<div class="card">
+		<header id="mainHeader" class="mainHeader">
+			<h1>Shoei GT-Air</h1>
+		</header><!-- /header -->
+		<div class="genInfo">
+			<img src="images/helmetOne.jpg" alt="Product image">
+			<p>The Shoei GT-Air was developed using wind-tunnel testing at Shoei’s state-of-the-art facility in Japan which lead to an aerodynamically tuned tri-composite shell that mitigates buffeting, as well as wind and road noise. The plush removable liner is comfortable, quiet and removable when you need to wash it. Light, safe and comfortable, the Shoei GT-Air meets and exceeds the needs of contemporary riders, allowing them to enjoy what they love most…the ride.</p>
+			<div class="availability">
+				<h4>Available</h4>
+			</div>
+		</div>
+		<div class="specifications">
+			<header class="specHeader">
+				<h3>Size Chart</h3>
+			</header><!-- /header -->
+			<div class="specList">
+				<ul>
+					<li>Small:</li>
+					<li>20 7/8" - 21 1/4"</li>
+					<li>Medium:</li>
+					<li>22 1/2"- 22 7/8"</li>
+					<li>Large:</li>
+					<li>23 1/4" - 23 5/8"</li>
+				</ul>
+			</div>
+			<div class="specFooter">
+				<h6>Specifications valid until 4/01/16</h6>
+			</div>
+		</div>
+		<div class="pricing">
+			<header>
+				<h3>Pricing</h3>
+			</header><!-- /header -->
+			<ul class="priceList">
```

Comment:
The format of this pricelist is 👍 , but the specs say it should contain information about varied pricing per quantity